doc/source/examples/extended_examples/Krylov/krylov_example.rst

```diff
@@ -75,11 +79,10 @@ ten elements per wavelength.
     nelem_wl = 10                    # no of elements per wavelength
     tol_elem = nelem_wl * no_wl      # total number of elements across length
 
-Element and material definition
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Here we will assign fluid medium (Air) properties to the duct.
-We will use Fluid 220 (Keyopt(2)=1) with one degree of freedom per node : pressure,
```

Comment:
I wasn't sure about this construction: freedom per node : pressure, ....

So, please review my changes carefully.

Reply:
changed to : " with one degree of freedom per node (pressure),"